Update to latest presto 0.175

The interface has changed since 0.130

```
[ERROR] presto-teradata-connector/src/main/java/com/facebook/presto/plugin/teradata/TeradataClient.java:[239,5] method does not override or implement a method from a supertype
[ERROR] presto-teradata-connector/src/main/java/com/facebook/presto/plugin/teradata/TeradataClient.java:[243,21] method buildSql in class com.facebook.presto.plugin.jdbc.BaseJdbcClient cannot be applied to given types;
[ERROR] required: java.sql.Connection,com.facebook.presto.plugin.jdbc.JdbcSplit,java.util.List<com.facebook.presto.plugin.jdbc.JdbcColumnHandle>
[ERROR] found: com.facebook.presto.plugin.jdbc.JdbcSplit,java.util.List<com.facebook.presto.plugin.jdbc.JdbcColumnHandle>
```

--- a/pom.xml
+++ b/pom.xml
@@ -75,7 +75,7 @@
 	</repositories>
 
 	<properties>
-		<presto.version>0.130</presto.version>
+		<presto.version>0.175</presto.version>
 		<airlift.version>0.120</airlift.version>
 		<guava.version>18.0</guava.version>
 		<guice.version>4.0</guice.version>

--- a/src/main/java/com/facebook/presto/plugin/teradata/TeradataClient.java
+++ b/src/main/java/com/facebook/presto/plugin/teradata/TeradataClient.java
@@ -166,7 +166,7 @@ public class TeradataClient extends BaseJdbcClient {
                 boolean found = false;
                 while (resultSet.next()) {
                     found = true;
-                    Type columnType = toPrestoType(resultSet.getInt("DATA_TYPE"));
+                    Type columnType = toPrestoType(resultSet.getInt("DATA_TYPE"), resultSet.getInt("COLUMN_SIZE"));
                     // skip unsupported column types
                     if (columnType != null) {
                         String columnName = resultSet.getString("COLUMN_NAME");
@@ -237,10 +237,10 @@ public class TeradataClient extends BaseJdbcClient {
     }
 
     @Override
-    public String buildSql(JdbcSplit split, List<JdbcColumnHandle> columnHandles) {
+    public PreparedStatement buildSql(Connection connection, JdbcSplit split, List<JdbcColumnHandle> columnHandles) throws SQLException {
         log.info("We are debugging how sql is built!");//TODO delete
         log.info(String.format("[split]: %s", split));
-        return super.buildSql(split, columnHandles);
+        return super.buildSql(connection, split, columnHandles);
     }
 
     @Override

